tests/integration/test_keeper_four_word_command/test.py

```diff
@@ -183,8 +183,8 @@ def test_cmd_mntr(started_cluster):
         # contains:
         #   10 nodes created by test
         #   3 nodes created by clickhouse "/clickhouse/task_queue/ddl"
-        #   1 root node, 3 keeper system nodes
```

Comment:
I probably need to change this back and add test in the test_keeper_availability_.py instead.

Reply:
done.